agoric/contract/package.json

```
@@ -47,6 +47,7 @@
     "@endo/far": "^0.2.18",
     "@endo/init": "^0.5.56",
     "@endo/marshal": "^0.8.5",
+    "@endo/patterns": "^0.2.6",
```

```suggestion
    "@endo/patterns": "^0.2.2",
```

is what is on mainnet, let's keep things consistent